hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -645,4 +625,55 @@ public void setMetaPath(String metaPath) {
   public void setActiveTimeline(HoodieActiveTimeline activeTimeline) {
     this.activeTimeline = activeTimeline;
   }
+
```

Comment:
IMO, if we have a **static** method, like this:

```java
public static Builder builder() {
        return new Builder();
}
```

Then, we can use it via this style:

```
HoodieTableMetaClient.builder().setXXX().setXXX()....build();
```

It will make the call chain more readable. wdyt?

Reply:
done.